src/schema/__tests__/ecommerce/order_fields.ts

```diff
@@ -0,0 +1,135 @@
+import gql from "lib/gql"
+
+export const OrderBuyerFields = gql`
```

Comment:
Once we have GraphQL intellisense setup in MP at some point, this won’t really work as it’s not valid GraphQL syntax (does the syntax highlighter work?). You can fix that by making these inline fragments:

```gql
... on Order {
  id
  code
  # etc
}
```